### PR TITLE
Toggle background image

### PR DESCRIPTION
**Pull request by FreeLikeGNU**
Toggle the background image from preferences

### CODE REVIEW

File: src/components/PlayerHome.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const PlayerHome = Vue.component("player-home", {
     },
     template: `
         <div id="player-home">
+          <div class="background">
```
Comment (alrusdi) [inline src/components/PlayerHome.ts:59]: Why do we need additional tag her?

File: src/styles/player_home.less [hunk 2]
```diff
@@ -6,6 +6,9 @@
 #player-home {
     padding-left: 20px;
     border-left: 1px solid #adacac;
+}
+
+.background {
```
Comment (alrusdi) [inline src/styles/player_home.less:11]: This class seems to be not needed

File: src/styles/preferences.less [hunk 3]
```diff
@@ -91,3 +91,9 @@
     margin-left: 38px;
  }
 
+/* Remove background */
+.preferences_remove_background .background {
```
Comment (alrusdi) [inline src/styles/preferences.less:95]: Why not just:
#player-home.preferences_remove_background
?

Comment (FreeLikeGNU) [reply src/styles/preferences.less]: that did not seem to work but:
`.preferences_remove_background #player-home`

File: src/styles/player_home.less (hunk 2 above)
Comment (FreeLikeGNU) [reply src/styles/player_home.less]: requested changes committed, thank you!

File: src/components/PlayerHome.ts (hunk 1 above)
Comment (FreeLikeGNU) [reply src/components/PlayerHome.ts]: requested changes committed, thank you!

**Pull request merged**